apis/config/v1beta1/configuration_types.go

```diff
@@ -82,6 +82,14 @@ type ControllerManager struct {
 	// +optional
 	Health ControllerHealth `json:"health,omitempty"`
 
+	// PprofBindAddress is the TCP address that the controller should bind to
+	// for serving pprof.
+	// It can be set to "" or "0" to disable the pprof serving.
+	// Since pprof may contain sensitive information, make sure to protect it
```

Comment:
Since we are not exposing this port through a Service, it can only be used if you ssh into node to attach to the port, right?

Did you have a chance to test it?
Can you include some documentation? Although that can be done in a separate PR.

Reply:
In my mind, usually `port-forward` command is enough to reach the port of pod to surf web part (or cli) of pprof or even to take a dump. For security reasons it's better nothing to expose out of the cluster. #983

Reply:
ah yes, port-forward seems reasonable.

/approve
/lgtm